### PR TITLE
feat added landing Page without node-modules

### PR DESCRIPTION
**Pull request by ganesh-kanchi**
Here is the deployment preview:
https://deploy-preview-2--jacket-ex-react.netlify.app/

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Overall the code looks fine

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Home } from "./pages/Home/Home";
 
 function App() {
```
Comment (mariyasada) [inline src/App.js:4]: create a separate file and store all the pages In that

**Review by mariyasada — APPROVED** [review]
This code looks good to me. good to merge

File: src/App.js (hunk 1 above)
Comment (ganesh-kanchi) [reply src/App.js:4]: I thought this would be cleaner.

**Pull request merged**